devops/scripts/create_esrp_job.ps1

```diff
@@ -0,0 +1,87 @@
+# create_esrp_job.ps1
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+<#
```

Comment:
Remove this line. License is stated at root of project for all files. Keep just the copyright line.